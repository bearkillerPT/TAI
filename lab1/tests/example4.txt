a aa ab ac b ba bb bc