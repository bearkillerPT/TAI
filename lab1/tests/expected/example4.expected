{'a': {' ': {'a': 2, 'b': 0, ' ': 0}, 'a': {' ': 1, 'b': 0, 'a': 0}, 'b': {' ': 1, 'b': 0, 'a': 0}}, ' ': {'a': {'a': 1, 'b': 2, ' ': 0}}, 'b': {' ': {'a': 1, 'b': 0, ' ': 0}}}